core/src/lib/cli.cc

```diff
@@ -142,12 +143,43 @@ void InitCLIApp(CLI::App& app, std::string description, int fsfyear)
 #ifdef HAVE_WIN32
   app.allow_windows_style_options();
 #endif
+  app.failure_message(CLI::FailureMessage::help);
+}
+
+void AddDeprecatedExportOptionsHelp(CLI::App& app)
+{
+  app.add_option(
+         "-x",
+         [&app](std::vector<std::string> vals) {
+           app.failure_message(CLI::FailureMessage::simple);
+           throw CLI::ParseError(
+               "The -xc and -xs options have changed. Use --xc and --xs as "
```

Comment:
I would like to suggest the following format for the syntax change hints:
```
The -dt option has changed.                                                        
Use --dt as given in the help.                                                     
                       
Run with --help for more information.
```
This way, the old and new syntax is printed one below the other and it is easier to see the difference.